client/android/divkit-demo-app/src/main/java/com/yandex/divkit/demo/div/DemoCustomContainerAdapter.kt

```diff
@@ -110,5 +110,8 @@ class DemoCustomContainerAdapter: DivCustomContainerViewAdapter {
             }
         }
         start()
+        this.postDelayed({
+            stop()
+        }, 3000)
```

Comment:
Added time stop after 2 seconds of ticking to create proper screenshots in interactive tests of customcard (without it test may be flaky, because time spend on screenshot differs from time to time)